CHANGELOG.md

```
@@ -1,15 +1,25 @@
 
-#### 9.4.19 - SNAPSHOT
+#### 9.5.3 - November 2024
   * (PW-2040) Updated the BBAN validation data file to the IBAN REGISTRY Jul 2024 release 
+  * (PW-2006) Fixed `getMUR` and `setMUR` in `SwiftMessage` to prioritize field 108 in block 4 over block 3 for system messages (category 0)
+  * Added new `MtSequenceEnum` with all the available inner sequences of specific MT schemas
+  * Added `isSystemMessage()` to SwiftMessage to check if the message is a category 0 message (010. 011, etc...)
   * Added new `MtSequenceEnum` with all the available inner sequences of specific MT schemas
 
-#### 9.4.18 - October 2024
+#### 9.5.2 - October 2024
   * Added new `FieldEnum` with all the available field names
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)